grammar.js

```diff
@@ -584,12 +579,12 @@ module.exports = grammar({
     ),
 
     _dml_read: $ => seq(
-      choice(
-        seq(
-          optional(
-            $._cte
-          ),
+      optional($._cte),
+      optional_parenthesis(
+        choice(
           $._select_statement,
+          $.union,
+          $.intersection,
```

Comment:
there's also an `except` set operation that yields all tuples in a not present in b

Reply:
I had merged that with `intersection` should these all be grouped under a common `set_operation` node instead?

Reply:
oh I like having the generic node type discriminable by the `operation` field in the third!